inst/include/lib/NiftiImage.h

```diff
@@ -453,6 +489,8 @@ class NiftiImage
     **/
     NiftiImage & reorient (const int i, const int j, const int k);
     
+
+#ifndef _NO_R__
```

Comment:
This should probably be moved down to fence out only the `update` function. As mentioned below, `reorient` shouldn't cause problems, and the string variant is quite convenient.